react-native/react/menubar/index.render.desktop.js

```diff
@@ -1,50 +1,78 @@
+const publicFolder = `file:///${resolveAssets('../react-native/react/images/folders/fa-folder-public-empty.svg')}`
+const privateFolder = `file:///${resolveAssets('../react-native/react/images/folders/fa-folder-private-empty.svg')}`
 
 export default class Render extends Component {
   props: {
+    username: ?string,
+    openingMessage: ?string,
+    openKBFS: () => void,
+    openKBFSPublic: () => void,
+    openKBFSPrivate: () => void,
+    showMain: () => void,
+    showHelp: () => void,
+    quit: () => void,
+    openingButtonInfo: {
+      text: string,
+      onClick: () => void
+    },
     debug?: boolean
   };
 
   render (): ReactElement {
+    const {openKBFS, openKBFSPublic, openKBFSPrivate, showMain, showHelp, quit, openingButtonInfo} = this.props
+
     return (
-      <div>
-        <Header/>
-        <OpeningMessage/>
-        <FolderList/>
-        <Footer debug={this.props.debug || false}/>
+      <div style={{backgroundColor: '#f6f6f4', width: 320}}>
+        <Header openKBFS={openKBFS} showHelp={showHelp}/>
+        {this.props.openingMessage && <OpeningMessage message={this.props.openingMessage} buttonInfo={openingButtonInfo} showHelp={showHelp}/>}
+        {this.props.username && <FolderList username={this.props.username} openKBFSPublic={openKBFSPublic} openKBFSPrivate={openKBFSPrivate}/>}
+        <Footer debug={this.props.debug || false} showHelp={showHelp} quit={quit} showMain={showMain}/>
       </div>
     )
   }
 }
 
 class Header extends Component {
+  props: {
+    openKBFS: () => void,
+    showHelp: () => void
+  };
+
   render () {
     return (
-      <div>
-        <span>FOLDER</span>
-        <span>website</span>
-        <span>usage</span>
+      <div style={{display: 'flex', flexDirection: 'row', paddingTop: 11, paddingLeft: 11, paddingBottom: 8}}>
+        <i className={`fa fa-folder`} style={{...Clickable, color: '#777777', fontSize: 16, marginRight: '10px'}} onClick={this.props.openKBFS}/>
+        <i className={`fa fa-globe`} style={{...Clickable, color: '#777777'}} onClick={this.props.showHelp}/>
       </div>
     )
   }
 }
 
 class OpeningMessage extends Component {
+  props: {
+    message: string,
+    buttonInfo: ?{
+      text: string,
+      onClick: () => void
+    }
+  };
+
   render () {
+    const {buttonInfo} = this.props
     return (
-      <div>
-        <div> Success! Some cool message goes here on two lines. </div>
+      <div style={{backgroundColor: '#00bff0', display: 'flex', flexDirection: 'column', alignItems: 'center'}}>
+        <div style={{textAlign: 'center', color: '#ffffff', marginTop: 18, marginBottom: 11, marginRight: 32, marginLeft: 32}}>{this.props.message}</div>
         <div>
-          <FlatButton style={commonStyles.primaryButton} label='getStarted'/>
+          {buttonInfo && <FlatButton style={{...commonStyles.primaryButton, ...NotoSans, fontSize: 17, marginBottom: 22}} onClick={buttonInfo.onClick} label={buttonInfo.text}/>}
```

Comment:
font size is fine (but maybe we should talk to design) but i think noto sans should get pickedup. i think its applied at a top level in style.css. maybe thats not being loaded in the remote components?